lib/rules/checked-requires-onchange-or-readonly.js

```diff
@@ -24,7 +24,7 @@ const defaultOptions = {
 };
 
 /**
- * @param {string[]} properties
+ * @param {Object[]} properties
```

Comment:
```suggestion
 * @param {object[]} properties
```
let's use TS-jsdoc since that's the one that matters

Reply:
Thanks! Fixed in 844d483